packages/config/src/chainConfig/types.ts

```diff
@@ -47,4 +48,70 @@ export interface IChainConfig {
   DEPOSIT_CHAIN_ID: number;
   DEPOSIT_NETWORK_ID: number;
   DEPOSIT_CONTRACT_ADDRESS: Uint8Array;
-}
+};
+
+export const chainConfigTypes: SpecTypes<IChainConfig> = {
+  PRESET_BASE: "string",
```

Comment:
should these values be enum? or overkill?

Reply:
Overkill? This whole approach will be review soon anyway to make parsing im the validator client and cli more resilient